packages/create/index.js

```diff
@@ -152,19 +167,21 @@ workspace(
     managed_directories = {"@npm": ["node_modules"]},
 )
 
-# Install the nodejs "bootstrap" package
-# This provides the basic tools for running and packaging nodejs programs in Bazel
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-http_archive(
-    name = "build_bazel_rules_nodejs",
-    sha256 = "65067dcad93a61deb593be7d3d9a32a4577d09665536d8da536d731da5cd15e2",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/3.4.2/rules_nodejs-3.4.2.tar.gz"],
-)
+load("//tools:bazel_deps.bzl", "fetch_dependencies")
+
+fetch_dependencies()
 
 ${pkgMgr === 'yarn' ? yarnInstallCmd : npmInstallCmd}`;
 
-  write('WORKSPACE.bazel', workspaceContent);
-  write('.bazelignore', `node_modules
+  write('tools/BUILD.bazel', '# No targets in this Bazel package\n')
```

Comment:
nit: "No targets in this Bazel package" makes it seem to me that it's not valid to put targets in the root package, rather than saying that this package is currently empty?

Reply:
fixed